### DISCUSSION TITLE
Podomoro timer & more

### DISCUSSION THREAD

**Discussion by diegomics** [alainm23/planify#1622] [Ideas]
Hi Dev, congrats for this app! In a hardware-related departure from elementary OS, I came across a Windows app called "Focus To-Do" with a very handy feature I think super interesting for Planner: a Pomodoro timer that can be linked to the task you are working in. Check it out! ;)

**Comment by ruanformigoni** [comment]
The [superproductivity](https://snapcraft.io/superproductivity) app has this feature, it is indeed quite handy. There are statistics on hours spent on each project and the workday. I find this very useful to analyse my productivity.

**Comment by josehbez** [comment]
Hi! Everybody. 

I'm suggestion integrate with https://github.com/gnome-pomodoro is written in vala-lang too.

Each task add: 

1.- Start Focus
![image](https://user-images.githubusercontent.com/13144779/113905839-3468b580-9799-11eb-8a71-6ce01ec6cd02.png)

2.- Estimation Focus
![image](https://user-images.githubusercontent.com/13144779/113905971-57936500-9799-11eb-8a4d-41df08c4e149.png)

3.- Simple stats 
![image](https://user-images.githubusercontent.com/13144779/113906186-988b7980-9799-11eb-801f-b560e7f89d81.png)

**Comment by sramkrishna** [comment]
I think this would be a great idea. Could be a great plugin.
